ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/tracking/webcam/WebCamUtils.kt

```diff
@@ -31,7 +31,7 @@ fun collectAllDevices(): List<WebCamInfo> {
 
         if (!camera.isOpened || !makePhoto(camera, frame)) {
             camera.release()
-            continue
+            break
```

Comment:
Probably it it better to use do..while loop, in theory the user can have more than MAX_CAMERA_QUANTITY devices